README.md

```diff
@@ -27,7 +27,7 @@ sudo modprobe cfg80211
 
 Insert `vwifi` driver:
 ```shell
-sudo insmod vwifi.ko sdssids='MyWifi+MyWifi_2+MyHomeWiFi'
+sudo insmod vwifi.ko ssid_list='MyWifi+MyWifi_2+MyHomeWiFi'
```

Comment:
Why did you choose character `+` as a delimiter?

Reply:
You should document the delimiters in top-level `README.md` as well.

Reply:
Because `+` sign is one of the forbidden char in SSID.

Reply:
According to the standard (802.11-2012) the SSID can be 0 to 32 octets in length. The encoding can be UTF-8 or Unspecified. However, some routers/firmware would disallow some characters.
See https://www.cisco.com/assets/sol/sb/WAP321_Emulators/WAP321_Emulator_v1.0.0.3/help/Wireless05.html

You should mention the limitation and usage.
How about specifying SSID like `[ThisIsSSID1][ThisIsSSID2][ThisIsSSID3]`?

By the way, you have to validate the length in advance.

Reply:
Well in this case it'll be hard to deal with if `[]` are in the SSIDs though i haven't thought of a better and elegant solution yet

Reply:
Using character `+` as the delimiter may result in crowded and ugly strings. And some companies even names themselves with `+`. e.g., http://www.aplusproducts.net/about/
In the later case, there might be `A+ Group` used in SSID name.

Reply:
OK. 
What do you think if we split it with four spaces ?

Reply:
Don't use spaces as delimiters -- it is hard to write down correct scripts for automation.

Reply:
How about `[MySSID1],[MySSID2],[MySSID3],` ?
There'll be less collision if it ends with `],`

Reply:
tried implement it